content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -85,6 +85,11 @@ to manage the lifecycle of Istio.
     {{< text bash >}}
     $ helm init --service-account tiller
     {{< /text >}}
+    
+1. Install CRDs:
+    {{< text bash >}}
+    $ kubectl apply -f install/kubernetes/helm/istio/templates/crds.yaml
```

Comment:
Surround the file path with @@ in order to generate a link to the yaml file in GitHub.